extensions/memory-operands-2/README.md

```diff
@@ -49,6 +49,10 @@ The behavior when `REX.Q` is set while in `quad` addressing mode when _any_ of t
   - The instruction has an `iS` and a `d8`
   - The instruction has a `dP` and an `i8`
 
+## Exceptions
```

Comment:
I'm generally hesitant to overload the term "Exceptions" since hardware/ISA exceptions are different from natural-language exceptions. These instructions don't cause exceptions, discounting memory access faults inherent to the memory operand, which isn't what is discussed in this section. Maybe something like "Unaffected Instructions"?

Reply:
Understandable. I removed the exceptions section